### ISSUE TITLE
The feasibility of outputs keypoint and descriptor

### ISSUE THREAD

**Issue by aruba01** [qianqianwang68/caps#9]
I am doing some work on visual localization. I think that this task needs tailored keypoint and descriptor and CAPS can be seen as tailored. As Superpoint output keypoint and descriptor simultaneously, can the future work of CAPS do the same thing?

**Comment by qianqianwang68** [comment]
Yes, for sure. Jointly learning keypoint and descriptors would be a great idea. [DISK](https://proceedings.neurips.cc/paper/2020/file/a42a596fc71e17828440030074d15e74-Paper.pdf) has shown that the epipolar constraints can be used to learn keypoints as effectively as using strong supervision of depths. We haven't explored the joint keypoint and descriptor training, but it would be an interesting and important future direction.
